packages/charts/src/styles/vaadin-chart-base-styles.js

```diff
@@ -532,11 +557,10 @@ export const chartStyles = css`
 
   :where([styled-mode]) .highcharts-funnel-series .highcharts-point {
     stroke-linejoin: round;
-    stroke: var(--_point-border);
-    stroke-width: 2px;
+    stroke: var(--highcharts-background-color, var(--_point-border));
   }
 
-  :where([styled-mode]) .highcharts-funnel-series .highcharts-point-hover {
+  :where([styled-mode]) .highcharts-funnel-series .highcharts-point-over {
```

Comment:
Is this change intended? Other places seem to use `hover`.

Reply:
An accident while editing the file, most likely.

```suggestion
  :where([styled-mode]) .highcharts-funnel-series .highcharts-point-hover {
```